code/game/objects/items/explosives/grenades/bullet_grenade.dm

```diff
@@ -51,3 +51,19 @@
 	fire_sound = 'sound/weapons/burst_phaser2.ogg'
 	projectile_count = 20
 	ammo_type = /datum/ammo/energy/lasburster
+/obj/item/explosive/grenade/bullet/hefa
+	name = "\improper M25 HEFA grenade"
+	desc = "High explosive fragmentation grenades cause a powerful yet very small explosion combined with a scattering ring of buckshot shrapnel, please throw very, very, VERY far away."
+	icon_state = "grenade_hefa2"
+	item_state = "grenade_hefa2"
+	icon_state_mini = "grenade_hefa"
+	hud_state = "grenade_hefa2"
+
+	rotations = -1
+	fire_sound = null
+	projectile_count = 10
+	ammo_type = /datum/ammo/bullet/hefa_buckshot
+
+/obj/item/explosive/grenade/bullet/hefa/prime()
+    explosion(loc, light_impact_range = 2, heavy_impact_range = 1)
+    return ..()
```

Comment:
According to linters you used spacebars on 68 and 69 instead of using tab.
Delete these spaces and put in a proper indent instead.